src/data/events.json

```diff
@@ -0,0 +1,6 @@
+[
```

Comment:
To remove or change?

Reply:
With the meetup search API I was'nt able to find all API Platform meetups (their search API does not support space). For now I'm just getting our "API Platform group" meetups. So I added this JSON file so that @CecileAmrl can add all the meetups she wants to highlight.